init.el

```diff
@@ -119,7 +128,8 @@ Ideally, this will be ~/.emacs.d.")
                          (global-set-key (kbd "M-y") 'helm-show-kill-ring)
                          (global-set-key (kbd "C-x c SPC") 'helm-all-mark-rings)
                          (global-set-key (kbd "C-h SPC") 'helm-all-mark-rings)
-                         (global-set-key (kbd "C-x c r i") 'helm-register)))
+                         (global-set-key (kbd "C-x c r i") 'helm-register)
+                         (global-set-key (kbd "M-i") 'helm-imenu)))
          (:name helm-ag
```

Comment:
I would prefer if this key-binding also lives within the same scope as other `helm-` key-bindings (i.e. `C-x c ...`). Also, I think that binding `helm-semantic-or-imenu` is slightly more useful. Can you change this to `C-x c i` for `helm-semantic-or-imenu` ?